implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/types/DynatraceSummary.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2022 VMware, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.dynatrace.types;
+
+/**
+ * Internal class for resettable summary statistics
+ *
+ * @author Georg Pirklbauer
+ * @since 1.9.0
+ */
+final class DynatraceSummary {
+    private long count = 0;
```

Comment:
There are better tools than `syncronized` to make this class thread safe, could you please check implementations of `DistributionSummary`/`AbstractDistributionSummary`?
e.g.: Check `StepDistributionSummary`, I guess you need something similar but with a reseting max/min.

You can use `LongAdder` for `count`, `DoubleAdder` for `total` and `AtomicLong` for `min` and `max`.